### PR TITLE
Fix for Regression failures

### PR DESCRIPTION
**Pull request by praveenavas**
Fix for Regression failures

### CODE REVIEW

File: cypress/e2e/WebInterface/Measure/EditMeasure/EditMeasureOwnershipValidations.cy.ts [hunk 1]
```diff
@@ -36,9 +36,13 @@ describe('Read only for measure, measure group, and test cases that user does no
         OktaLogin.Login()
     })
 
-    afterEach('Logout', () => {
+    afterEach('Logout and clean up', () => {
+
+        //Log out
```
Comment (cvasile001) [inline cypress/e2e/WebInterface/Measure/EditMeasure/EditMeasureOwnershipValidations.cy.ts:41]: should we enable logout in this case? is this related to the cookie issue?

**Review by cvasile001 — CHANGES_REQUESTED** [review]
see my question

File: cypress/e2e/WebInterface/Measure/EditMeasure/EditMeasureOwnershipValidations.cy.ts (hunk 1 above)
Comment (praveenavas) [reply cypress/e2e/WebInterface/Measure/EditMeasure/EditMeasureOwnershipValidations.cy.ts:41]: yes this is related to cookie issue. Added new function for logout.

**Review by cvasile001 — APPROVED** [review]
approved
